src/Polly.Core/Utils/ObjectPool.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Concurrent;
+using System.Threading;
+
+namespace Polly.Utils;
+
+// copied from https://raw.githubusercontent.com/dotnet/aspnetcore/main/src/ObjectPool/src/DefaultObjectPool.cs
```

Comment:
Let's update this to point at a specific Git SHA so we can tell _when_ the code was taken too.

Reply:
Done.